k-distribution/tutorial/1_k/2_imp/lesson_5/imp

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 export PATH=$PATH:"$(dirname "$0")"/../../../../bin
-llvm-krun -d "$(dirname "$0")/imp-kompiled" -c PGM "$1" Pgm prettyfile -p
+pgm=$1
+shift
+llvm-krun -c PGM "$pgm" Pgm prettyfile -p "$@" -d "$(dirname "$0")/imp-kompiled"
```

Comment:
This is a little confusing to me as an example, because it looks like `$@` is maybe all arguments to `-p`? Or no?

If not, can you put it at the end in a position where it's less confusing as an example?

```suggestion
llvm-krun -c PGM "$pgm" Pgm prettyfile -p -d "$(dirname "$0")/imp-kompiled" "$@"
```

Reply:
I would but then if you pass -d to `imp` as the K makefile does, the script will break.